From 9d18d27a993547e017355b31eebbd4ba6ef8cf78 Mon Sep 17 00:00:00 2001
From: Quynh Nguyen <quynh.nguyen.xb@renesas.com>
Date: Thu, 12 Apr 2018 00:18:51 +0700
Subject: [PATCH 456/475] usb: renesas_usbhs: Add support for R8A7744

This driver supports other SoCs, but they need boards/Soc depend code.
So, this patch adds device tree support for R8A7744 (RZ/G1N) SoC.

Signed-off-by: thongsyho <thong.ho.px@rvc.renesas.com>
Signed-off-by: Quynh Nguyen <quynhnguyen.xb@renesas.com>
---
 drivers/usb/renesas_usbhs/common.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/usb/renesas_usbhs/common.c b/drivers/usb/renesas_usbhs/common.c
index 54683fd..6ee38b5 100644
--- a/drivers/usb/renesas_usbhs/common.c
+++ b/drivers/usb/renesas_usbhs/common.c
@@ -477,6 +477,10 @@ static const struct of_device_id usbhs_of_match[] = {
 		.data = (void *)USBHS_TYPE_RCAR_GEN2,
 	},
 	{
+		.compatible = "renesas,usbhs-r8a7744",
+		.data = (void *)USBHS_TYPE_RCAR_GEN2,
+	},
+	{
 		.compatible = "renesas,usbhs-r8a7794",
 		.data = (void *)USBHS_TYPE_RCAR_GEN2,
 	},
-- 
2.7.4

